addOns/encoder/src/test/java/org/zaproxy/addon/encoder/processors/predefined/utility/AscifyUnitTest.java

```diff
@@ -42,4 +42,13 @@ void shouldEncodeWithoutError() throws Exception {
         assertThat(result.hasError(), is(equalTo(false)));
         assertThat(result.getResult(), is(equalTo("This is a funky String: fi. etrange.")));
     }
+
+    @Test
+    void mayRemoveAllChars() {
+        // Given / When - According to deepl.com this is 'bird' in simplified Chinese
```

Comment:
Can we drop the reference? Do we care what the word is? Looks good anyway.

Reply:
I guess we can.

I added it because the first thing I found looking around translated to 'foolish chicken' or something like that 😉 (after that I didn't want to find out I insulted someone 3 years from now)

Reply:
wiktionary allows to double check that.